### ISSUE TITLE
Fix the flaky test padl.creator.classfile.test.visitor.Composite1

### ISSUE THREAD

**Issue by yann-gael** [ptidejteam/ptidej-Ptidej#7]
The test `padl.creator.classfile.test.visitor.Composite1` is flaky: sometimes is passes, sometimes it fails without clear reasons.  Fix/Change it so that it is consistent.

**Comment by LVoisard** [comment]
I wasn't able to fix this issue or find what causes it yet but here is what I found out:

- I was able to reproduce the problem magically by editing the expected.txt in resources and reversing my changes, making it the same as it used to be, but now the tests failed.
- I then fixed the issue by doing a clean install of the entire project (not just PADL Analysis).

Next time the issue happens, I suggest looking at PADL Creator Classfiles, since the generator uses this module for the visitor test.


We also investigated whether the problem was an issue of CR LF however it was not the case
